sources/assets/Stride.Core.Assets.Quantum/Stride.Core.Assets.Quantum.csproj

```diff
@@ -8,6 +8,8 @@
     <StrideAssemblyProcessor>true</StrideAssemblyProcessor>
     <StrideAssemblyProcessorOptions>--auto-module-initializer --serialization</StrideAssemblyProcessorOptions>
     <RestorePackages>true</RestorePackages>
+    <UseWindowsForms>true</UseWindowsForms>
```

Comment:
Could the removal of these dependencies be preserved?

This does not appear to be the reasons why these changes where causing issues.

Reply:
We'll revert the revert once the PRs to fix is in, no need to faff around with individual lines